packages/mui-envinfo/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/envinfo",
-  "version": "2.0.11",
+  "version": "2.0.12",
```

Comment:
Are we updating this one?

Reply:
Yes, we are updating all public packages.